input-profile.js

```diff
@@ -0,0 +1,194 @@
+import {Component, Type} from '@wonderlandengine/api';
+import {vec3, quat} from 'gl-matrix';
+
+export class InputProfile extends Component {
+    static TypeName = 'input-profile';
+    static Properties = {
+        controller: {type: Type.Object},
```

Comment:
Use the new property type please:

```js
import {Property} from '@wonderlandengine/api';

export class InputProfile extends Component {
    static Properties = {
        controller: Property.object()
    };
}
```

Reply:
Nice , was not at all aware of this new change.